common/log.c

```diff
@@ -592,7 +592,7 @@ log_message(const enum logLevels lvl, const char *msg, ...)
         /* log to syslog*/
         /* %s fix compiler warning 'not a string literal' */
         syslog(internal_log_xrdp2syslog(lvl), "(%d)(%ld)%s", g_getpid(),
-               tc_get_threadid(), buff + 20);
+               (long) tc_get_threadid(), buff + 20);
```

Comment:
Thread ID should fit `long` on any supported OS, even on Win64. I think the fix belongs to the tc_get_threadid() return type.